common/app/templates/systemJsSetup.scala.js

```diff
@@ -23,8 +23,8 @@ System.config({
 
 @JavaScript(Static.js.systemJsAppConfig)
 
-@if(!play.Play.isDev()) {
-    @JavaScript(Static.js.systemJsBundleConfig)
+@if(!play.Play.isDev() || (play.Play.isDev() && useBundlesInDev)) {
+    System.config({ bundles: @JavaScript(Static.js.systemJsBundleConfig) });
```

Comment:
I can't make this change because the head you pass along to main is inserted before System itself.